recipe/meta.yaml

```diff
@@ -1,8 +1,9 @@
 {% set clang_variant = os.environ.get('CLANG_VARIANT', 'flang') %}
-{% set flang_commit = "ab1423f97274b472970efd53e01800a6d4f137f5" %}
-{% set version = "5.0.0" %}
-{% set sha256 = "01264da25bac23e27ca9298bbfd41f7226854247e92a3edea6f28decf42c7b09" %}
-{% set build_number = "1" %}
+{% set flang_commit = "528304c136d60fd54f009f9c8533015bc23411b1" %}
+{% set version = "2017.12.14" %}
```

Comment:
When `flang` gets a release this would be problematic as these builds will be "newer" than those. How about setting build_number to be "2017.12.14" ?

Reply:
ping @xoviat

Reply:
Or, set the version to "0.2017.12.14"